flight/Modules/Stabilization/stabilization.c

```diff
@@ -568,6 +568,10 @@ static float calculate_thrust(StabilizationDesiredThrustModeOptions mode,
 			vertspeed_desired = desired_thrust;
 			break;
 
+		case STABILIZATIONDESIRED_THRUSTMODE_FLIPOVERMODETHRUSTREVERSED:
```

Comment:
Negate thrust, so that positive throttle produces negative thrust in the "reversed" mode.  This is what most people will want.  In a true 3D setup (with symmetric throttle) this is not necessary and the non-reversed mode can be used.

Positive thrust with flipover mode may be valuable, for cases where the quad is kinda pinned right-side-up.